src/internal/modules/charge-information/forms/charge-category/default-radio-options-form.js

```diff
@@ -39,7 +39,14 @@ const form = request => {
 
   const f = formFactory(action, 'POST');
 
-  f.fields.push(fields.radio(stepKey, getChoices(config), data[stepKey] || ''));
+  if (config.caption) {
+    f.fields.push(fields.paragraph(null, {
+      text: config.caption,
+      controlClass: 'govuk-hint'
+    }));
+  }
+  const value = data[stepKey];
+  f.fields.push(fields.radio(stepKey, getChoices(config), value === undefined ? '' : value));
```

Comment:
Nope, because that will blanks out a value of false (which was the bug I was fixing).  We need to be able to show if it's set to true or false or unset.  The fix does that.